udata/tests/api/test_dataservices_api.py

```diff
@@ -61,18 +61,19 @@ def test_dataservice_api_create(self):
         self.assertEqual(response.json["base_api_url"], "https://example.org")
         self.assertEqual(response.json["tags"], ["hello", "world"])
         self.assertEqual(response.json["private"], True)
-        self.assertEqual(response.json["datasets"][0]["title"], datasets[0].title)
-        self.assertEqual(response.json["datasets"][1]["title"], datasets[2].title)
-        self.assertEqual(
-            response.json["extras"],
-            {
-                "foo": "bar",
-            },
-        )
+        self.assertEqual(response.json["extras"], {"foo": "bar"})
         self.assertEqual(response.json["license"], license.id)
         self.assertEqual(
             response.json["self_api_url"], "http://local.test/api/1/dataservices/updated-title/"
         )
+
+        self.assertEqual(response.json["datasets"]["total"], 2)
+        response_datasets = self.get(response.json["datasets"]["href"])
```

Comment:
Is it on purpose that you test the "by link" feature on the test PATCH response instead of a POST? Anything special about the PATCH?
Would it make sense to also test it on the POST response?

Reply:
It could be tested on either one, since both requests returns the same object structure. I tested it here because it was here that I tested that the datasets are correctly modified.

Reply:
Understood thanks, it makes sense, thanks for the explanation 🙏